src/main/java/training/endava/skeleton/model/PersonRepository.java

```diff
@@ -0,0 +1,75 @@
+package training.endava.skeleton.model;
+import training.endava.skeleton.SkeletonRepository;
+import training.endava.skeleton.exceptions.PersonIdAlreadyExistException;
+import training.endava.skeleton.exceptions.PersonIdNotFoundException;
+
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+public class PersonRepository implements SkeletonRepository<Person, Integer> {
+
+//    ArrayList<Person> persons= (ArrayList) MOCK_DB.getTable(Person.class);
+
+    List<Person> persons;
+
+    public PersonRepository(List<Person> persons) {
```

Comment:
You should use MOCK_DB to implement these CRUD operations. Please review them all and implement them trying to retrieve data from MOCK_DB.